.github/workflows/graphql-api-ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
-          node-version: '14.15.2'
+          node-version: '^14.17.0'
```

Comment:
If we're updating this, should we update the version of node used in the dockerfile? (`deploy/dockerfiles/browser/api.dockerfile`)

From my perspective, yes. But also, it currently looks like our frontend CI and frontend dockerfile have gotten out of sync, so I'm unsure if this was intentional or it happened at some point.

I would think it's useful to keep these in sync, so suggest updating the API dockerfile in this PR.

Reply:
Actually it turns out that the docker image won't build if they're out of sync, which, fair enough. Pushing changes shortly.